shared/src/packages/sites/libs/enums/site-validation-message.enum.ts

```diff
@@ -2,6 +2,8 @@ const SiteValidationMessage = {
   SITE_NAME_REQUIRE: 'Project name is required',
   SITE_NAME_WRONG: 'Project name is wrong',
   SITE_NAME_PATTERN: 'Incorrect project name',
+  SITE_TARGET_REQUIRE: 'Required field',
+  SITE_TARGET_WRONG: 'Invalid field',
```

Comment:
What field is required and what is invalid -- you have specific keys and abstract messages